### ISSUE TITLE
Move POD admin badge element outside of the <ul> element

### ISSUE THREAD

**Issue by ggeisler** [pod4lib/aggregator#842]
The POD admin badge we display when the user is a POD admin is implemented as a `<span>` element. 

<img width="359" alt="Screen Shot 2022-05-27 at 11 31 41 AM" src="https://user-images.githubusercontent.com/101482/170770861-4be1c99a-a5ca-4861-a636-70bf24f55f7f.png">

To place it in its current location next to the user link/select menu we've inserted that span into the `<ul>` that holds the user menu. 

<img width="483" alt="Screen Shot 2022-05-27 at 11 32 34 AM" src="https://user-images.githubusercontent.com/101482/170770883-ebe7a088-5003-4646-88af-90333dc46a83.png">

However, this placement violates an accessibility guideline that "Lists contain only `<li>` elements and script supporting elements (`<script>` and `<template>`)." 

We need to:

- [ ] Move the `<span>` element containing the badge outside of the `<ul>` element
- [ ] Maintain its current display position immediately adjacent to the user menu

This might involve experimenting a bit with flexbox, or perhaps there is an easier way.
